airbyte-integrations/connectors/source-open-exchange-rates/integration_tests/invalid_config.json

```
@@ -0,0 +1,5 @@
+{
+  "start_date": "2021-05-16",
+  "base": "USD",
+  "access_key": "wrong-access-key"
```

In your spec.yaml, you defined this key as `app_id`

```suggestion
  "app_id": "wrong-access-key"
```